tests/workspace-files.feature

```diff
@@ -0,0 +1,29 @@
+# Tests for the presence of files in the workspace
+
+Feature: Workspace files
+
+    As a researcher,
+    I want to make sure that my Snakemake workflow produces expected files,
+    and that the files fetched from the remote locations are not corrupted,
+    so that I can be sure that the workflow outputs are correct.
+
+    Scenario: The workspace contains all the relevant input files
+        When the workflow execution completes
+        Then the workspace should include "BuildFile.xml"
+        And the workspace should include "datasets/CMS_Run2011A_DoubleMu_AOD_12Oct2013-v1_10000_file_index.txt"
```

Comment:
You could also test for the presence of one scram file, for example:

```
CMSSW_5_3_32/.SCRAM/slc6_amd64_gcc472/timestamps/boost
```

Could be a new feature, to test whether scram works nicely. (Or part of testing for known "outputs" before final plots.)